### ISSUE TITLE
Request for Open-source Training Code for VAE

### ISSUE THREAD

**Issue by HuanyiZheng** [TencentARC/GeometryCrafter#6]
Hi,

We are trying to replicate the training process for your VAE model, and we were wondering if it would be possible to open-source the training code. Specifically, we are interested in the loss function for the VAE mentioned in your paper and would love to try it out.

Thank you for considering this request!
